moped-toolbox/populate_geometry_transform/transpose_geometry.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/python3
+#  docker run -it --rm -v "$(pwd)":/app geomtransform /bin/bash
 
 from operator import truediv
```

Comment:
Sometimes, I'll be working in vscode, and I'll typo something, and the IDE interpretes it as me trying to invoke a method from a library that I have not imported. I wish I knew which vscode plugin did this, I would turn it off. 

This import of `truediv`, which is a division operator that casts input integers into floats, is one of these. It can go, if you want. No harm in it though, if it stays.

Also, TIL `truediv`. `1/3 → 0` vs `operator.truediv(1,3) → 0.3̅`.

Reply:
🙏 that's absolutely what happend - +1 TIL :)